client/src/app/site/agenda/components/agenda-list/agenda-list.component.html

```diff
@@ -38,6 +38,9 @@
                 <div *ngIf="showSubtitle" class="subtitle ellipsis-overflow">
                     {{ item.getSubtitle() }}
                 </div>
+                <div *ngIf="item.comment" class="subtitle ellipsis-overflow">
+                    <os-icon-container size="small" icon="comment">{{ item.comment }}</os-icon-container>
+                </div>
```

Comment:
@emanuelschuetze not sure, but I suppose this should also depend on the "Show subtitles in the agenda" option.  (like above)